rskj-core/src/main/java/co/rsk/remasc/RemascTransaction.java

```diff
@@ -73,7 +73,7 @@ public long transactionCost(Constants constants, ActivationConfig.ForBlock activ
     }
 
     @Override
-    public RskAddress getSender() {
+    public synchronized RskAddress getSender() {
```

Comment:
Why we need to add this word? It would be better to remove the `synchronized` from parent class `Transaction` using an internal lock for BOTH `Transaction.getSender` methods

Reply:
Done!

Reply:
Just to keep it here, I think we need to talk about if its really necessary to have this synchronization here